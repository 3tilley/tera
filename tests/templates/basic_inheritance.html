{% extends "base.html" %}

{% block extrahead %}
  <script>
    console.log("Hello");
  </script>
{% endblock extrahead %}

{% block content %}
  <h1>{{ product.name }} - {{ product.manufacturer }}</h1>
{% endblock content %}
