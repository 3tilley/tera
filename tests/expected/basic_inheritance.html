<html>
  <head>
    <title>Moto G</title>
    
  </head>
  <body>
    <div>
  <h1>Moto G - Motorala</h1>
</div>
  </body>
</html>
